examples/basic/index.html

```diff
@@ -8,7 +8,7 @@
   <script src="https://unpkg.com/react-dom@15.3.1/dist/react-dom.js"></script>
   <script src="https://unpkg.com/prop-types/prop-types.min.js"></script>
   <!-- For local development remove the following line: -->
-  <script src="https://unpkg.com/react-google-button@latest/dist/react-google-button.js"></script>
+  <script src="react-google-button.js"></script>
```

Comment:
How does this work? Is it supposed to point to the one in the dist folder? It won't really work as a standalone demo then right?